CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+<!--
```

Comment:
please add about prerequisites, as in https://github.com/cdevents/.github/blob/main/docs/CONTRIBUTING.md#contributing-prerequisites-cladco . We can later discuss what to adopt in terms of tooling(from LFX tools)

Reply:
This has been added in the current push.